v1/ratelimit/redis_delayed_sync_pipelined.go

```diff
@@ -338,6 +344,13 @@ func (r *RedisDelayedSyncPipelined) processSyncRes(cmdArgs syncArgs, cmdRes inte
 		remote := vals[2].(int64)
 		cmdArgs.lmt.IncrementResetAtBy(diff)
 		r.lastSyncedResetAt.Store(key, remote)
+	case InitLocal:
+		remote := vals[1].(int64)
+		if remote > cmdArgs.lmt.GetResetAt() {
+			cmdArgs.lmt.SetResetAt(remote)
+			cmdArgs.lmt.PopResetAtDelta()
```

Comment:
Why do we need to pop this?

Reply:
I actually don't understand why it becomes meaningless because it is spent locally, but if reset to remote without adding it back, it becomes free.